Library/Formula/svtplay-dl.rb

```diff
@@ -25,11 +68,11 @@ class SvtplayDl < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    %w[requests].each do |r|
-      resource(r).stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
-      end
-    end
+    resources.each do |r|
+       r.stage do
+         system "python", *Language::Python.setup_install_args(libexec/"vendor")
+       end
+     end
 
```

Comment:
You need to add something like `touch libexec/"vendor/lib/python2.7/site-packages/ndg/__init__.py"` here for reasons with a long explanation explain (briefly, ndg is a namespace package and those are broken if they're installed outside of magical directories without this intervention)